src/Uno.UI/UI/Xaml/Controls/Layouter/Layouter.cs

```diff
@@ -105,7 +105,8 @@ public Size Measure(Size availableSize)
 				frameworkAvailableSize = frameworkAvailableSize
 					.Subtract(marginSize)
 					.AtLeastZero()
-					.AtMost(maxSize);
+					.AtMost(maxSize)
+					.AtLeast(minSize);
```

Comment:
To my memory it was an error to do that, it was in the original code but was causing problems. I don't remember exactly what it was.

Reply:
@carldebilly I don't think it is. I confirmed the behavior on Windows (and extended the existing test)
The real bug is few lines below (the use of `frameworkAvailableSize` instead of `availableSize`)

Reply:
That's possible. Anyway there's many tests around this. It will be important to thoroughly check the screenshots comparison for this PR.